packages/upload-api/src/blob/replicate.js

```diff
@@ -186,6 +187,9 @@ export const blobReplicateProvider = (context) => {
                 cause: invocation.cid,
               },
               facts: allocFacts,
+              // set the expiration now so that we get the same CID for the task
+              // when we call delegate/execute.
+              expiration: now() + 30
```

Comment:
Does this mean the receiving node has <= 30 seconds to act on the blob replicate invocation before it expires? If so, I am a bit concerned that isn't a lot enough time..

Reply:
No, this is just the time for it to receive and validate the invocation.